{{cookiecutter.plugin_name}}/pyproject.toml

```diff
@@ -13,7 +13,51 @@ write_to = "src/{{cookiecutter.module_name}}/_version.py"
 
 [tool.black]
 line-length = 79
+target-version = ['py38', 'py39', 'py310']
 
-[tool.isort]
-profile = "black"
-line_length = 79
+
+[tool.ruff]
+line-length = 79
+select = [
+    "E", "F", "W", #flake8
+    "UP", # pyupgrade
+    "I", # isort
+    "BLE", # flake8-blind-exception
+    "B", # flake8-bugbear
+    "A", # flake8-builtins
+    "C4", # flake8-comprehensions
+    "ISC", # flake8-implicit-str-concat
+    "G", # flake8-logging-format
+    "PIE", # flake8-pie
+    "SIM", # flake8-simplify
+]
+ignore = [
+    "E501", # line to long. left this on black side
+    "UP006", "UP007", # type annotation. As using magicgui require runtime type annotation then we disable this.
+    "SIM117", # flake8-simplify - some of merged with statements are not looking great with black, reanble after drop python 3.9
```

Comment:
```suggestion
    "SIM117", # flake8-simplify - some merged with statements are not looking great with black, reenable after dropping python 3.9
```

But why?

Reply:
This rule is copied from napari and bad example could be seen here: https://github.com/napari/napari/pull/5566#discussion_r1115066707